Src/Support/GoogleApis.Auth.Tests_dotnetcore/project.json

```diff
@@ -0,0 +1,37 @@
+﻿{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "Google.Apis.Core": "1.14",
+    "Google.Apis.Auth": "1.14",
+    "GoogleApis.Tests_dotnetcore": {
+      "target": "project"
+    },
+    "Nunit": "3.4.0",
+    "dotnet-test-nunit": "3.4.0-beta-1"
+  },
+
+  "compile": "../GoogleApis.Auth.Tests/**/*.cs",
+    
+  "exclude": [
+    "../GoogleApis.Auth.Tests/Program.cs",
+    "../GoogleApis.Auth.Tests/OAuth2/Flows/*.cs"
+  ],
+
+  "testRunner": "nunit",
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+	"netcoreapp1.0",
+	"portable-net45+win8"
```

Comment:
These are needed in order to be able to consume the nunit runner, see [this](http://www.alteridem.net/2016/06/18/nunit-3-testing-net-core-rc2/) as a reference.

These imports are for the tests project to consume nunit, not related to how it consumes the library being tested.